tutorials/clustering.md

```diff
@@ -13,17 +12,25 @@ In this tutorial, we will show how you can create a distributed hypertable withi
 a distributed database. We will set up the distributed database with one primary access node
 and two data nodes across three distinct TimescaleDB instances.
 
-Prerequisites:
-- 3 instances of TimescaleDB installed (private BETA branch only)
-- A superuser set up on all 3 instances of TimescaleDB
-- Network connectivity between the 3 TimescaleDB instances
-- Must be on PG11
+It is assumed before starting this tutorial you will have downloaded and installed the
+TimescaleDB multinode beta package on three networked machines running Postgres 11. It's
+also assumed you'll have access to some superuser role common to these three machines.
+
+Beyond this, please ensure that you've completed the following steps. Some or most of
+these may not be necessary depending on how you initially set up your instances (the 
+docker image should be have most of this already configured):
+- Make sure you've enabled 'max_prepared_transactions' on all the instances (set this to 100 if you're unsure), this parameter is located in `postgresql.conf`
```

Comment:
Unsure about what? Suggest to write "for example 100" and not in parentheses but rather commas.

Reply:
Yes, that felt clunky when I wrote it and I meant to come back and clean it up.  Still looked a bit funky as you suggested, so I added it as a second sentence here.